src/rgw/rgw_sal_motr.cc

```diff
@@ -549,11 +549,19 @@ int MotrUser::remove_user(const DoutPrefixProvider* dpp, optional_yield y)
   return 0;
 }
 
+static string get_bucket_name(string tenant, string bucket)
```

Comment:
Why not using references in arguments as was suggested?

Reply:
@andriytk it was throwing error - ` error: binding reference of type ‘std::__cxx11::string&’ {aka ‘std::__cxx11::basic_string<char>&’} to ‘const string’ {aka ‘const std::__cxx11::basic_string<char>’} discards qualifiers`

Can this be used?
```
static string get_bucket_name(std::string const& tenant, std::string const& bucket)
{
  if (tenant != "")
    return tenant + "$" + bucket;
  else
    return bucket;
}
```

Reply:
we can use it like
```c++
static string get_bucket_name(const std::string& tenant,  const std::string& bucket) {}
```
This is the concept of lval rval in c++.
`const &` means  the function can accept rval
`&` mean the function will accept on lval

Reply:
Yes, using `const` in args in fine, of course.

Reply:
Done